tools/run_tests/artifacts/artifact_targets.py

```diff
@@ -332,6 +347,10 @@ def targets():
            PythonArtifact('linux', 'x86', 'cp34-cp34m'),
            PythonArtifact('linux', 'x86', 'cp35-cp35m'),
            PythonArtifact('linux', 'x86', 'cp36-cp36m'),
+           PythonArtifact('linux', 'arm', '2.7'),
```

Comment:
can we have some tags so we can easily filter this out of the 'linux' build (`task_runner.py -f linux artifact`) once https://github.com/grpc/grpc/pull/11115 is merged?
perhaps we could use something like 'linux_extra' or 'linux_longrunning' instead of 'linux'?